src/bootloaders/shim-systemd.c

```diff
@@ -110,7 +111,7 @@ static char *shim_systemd_get_kernel_dst(const BootManager *manager) {
 
 static bool shim_systemd_install_kernel(const BootManager *manager, const Kernel *kernel) {
         fprintf(stderr, "Call %s\n", __func__);
-        return true;
+        return sd_class_install_kernel_impl(manager, kernel, shim_systemd_get_kernel_dst, NULL);
```

Comment:
So I guess I'm thinking about folks who would like to use Clear Linux without secure boot but perhaps that doesn't make any difference in how they will still be using shim and the rest of the new bits (I wasn't really thinking about that). If that's the case as long as our migration plan is going to not leave bits behind it is okay with me with your approach.